content/en/flux/installation/bootstrap/generic-git-server.md

```diff
@@ -90,6 +90,21 @@ You can also supply the password or Git token using a pipe e.g. `echo "<my-pass>
 If your Git server uses a self-signed TLS certificate, you can specify the CA file with
 `--ca-file=<path/to/ca.crt>`.
 
+## Bootstrap over HTTPS using Personal Access Token(PAT)
```

Comment:
```suggestion
## HTTPS authorization header
```

I think this tittle is better, usually PATs are used with basic auth, VBS is an exception to this.